crypto/rand/drbg_lib.c

```diff
@@ -412,25 +497,17 @@ static int drbg_bytes(unsigned char *out, int count)
 
 static int drbg_add(const void *buf, int num, double randomness)
 {
```

Comment:
Could you document this function, and say that randomness is ignored because it's used for additional data?

Reply:
I would like to add that normally we don't document those functions, because this one is documented by RAND_add()